myia/parser.py

```diff
@@ -129,6 +130,15 @@ def _fresh(node):
         return node
 
 
+def _eval_ast_node(node):
+    """Evaluate an AST node as a Python expression and return result."""
+    # eval() does not accept an AST node, so we first convert AST node
+    # to string code.
+    code = astunparse.unparse(node).strip()
+    # Then we parse it with eval().
+    return eval(code)
```

Comment:
We want to evaluate in the function's environment, though. Let's say we have this code, for example:

```python
from numpy import ndarray

def f(x: ndarray):
    return x * x
```

This is not going to work with just `eval(code)` because the symbol `ndarray` is not defined in parser.py. That's why we want to pass the function's globals (I think they would be in  `f.__globals__`) as the second argument to `eval`.